_rules/SC1-1-1-image-has-name.md

```diff
@@ -47,48 +47,74 @@ There is a known combination of a popular browser and assistive technology that
 
 #### Passed example 1
 
-Image has accessible name
+HTML `img` element has accessible name
 
 ```html
 <img alt="W3C logo" />
 ```
 
 #### Passed example 2
 
-Non-image element with image role and accessible name
+Element with role of `img` and accessible name
 
 ```html
 <div role="img" aria-label="W3C logo"></div>
 ```
 
 #### Passed example 3
 
-Accessible name but not always supported.
+Accessible name but not always accessibility supported
 
 ```html
 <img title="W3C logo" />
 ```
 
+#### Passed example 4
+
+HTML `img` element marked as [decorative](#decorative) through empty `alt` attribute
+
+```html
+<img alt="" />
+```
+
+#### Passed example 5
+
+HTML `img` element marked as [decorative](#decorative) through `role="presentation"`
+
+```html
+<img role="presentation" />
+```
+
+#### Passed example 6
+
+HTML `img` element marked as [decorative](#decorative) through `role="none"`
+
+```html
+<img role="none" />
+```
+
 ### Failed
 
 #### Failed example 1
 
-No accessible name
+No accessible name and is not marked as [decorative](#decorative)
 
 ```html
 <img />
 ```
 
 #### Failed example 2
 
-Non-image element with image role but no accessible name.
+Non-image element with image role but no accessible name
```

Comment:
To be consistent with the Passed exemple 2, change "Non-image element with image role"  by "Element with role of `img`"